### ISSUE TITLE
Testing fails on macOS Apple Chip - mysql:5.7 image build doesn't exist

### ISSUE THREAD

**Issue by lmilbaum** [hashicorp/vault#16061]
**Describe the bug**
Testing by invoking the command `make test` is broken on mac Apple chip.
`lmilbaum@Lioras-MacBook-Air-2 vault % docker pull mysql:5.7
5.7: Pulling from library/mysql
no matching manifest for linux/arm64/v8 in the manifest list entries`

**To Reproduce**
1. Run `docker pull mysql:5.7`

**Expected behavior**
`make test` should be passing for HEAD main

**Environment:**
macOS Monterey
Applie chip

**Comment by hsimon-hashicorp** [comment]
Hi @lmilbaum - is this a problem you're having with Vault? If so, can you please provide more details?

**Comment by lmilbaum** [comment]
> Hi @lmilbaum - is this a problem you're having with Vault? If so, can you please provide more details?

Yes. This is a problem I'm having. What additional information do you need? Will be happy to provide.

**Comment by lmilbaum** [comment]
> @lmilbaum Is this a problem with Vault specifically? The only thing I see mentioned is mysql. What other steps are you running to get to this error?

The problem is not Vault specific. It is related to the fact that I am using an ARM based host. 

The `make test` command is broken because it is using mysql:5.7 container image which doesn't have an ARM based build. Does that clarifies the problem?
